metecho/api/models.py

```diff
@@ -44,6 +46,15 @@
 )
 
 
+class SiteProfile(TranslatableModel):
+    site = models.OneToOneField(Site, on_delete=models.CASCADE)
+
+    translations = TranslatedFields(
+        name=models.CharField(max_length=64),
+        clickthrough_agreement=MarkdownField(property_suffix="_markdown", blank=True),
+    )
```

Comment:
This is obviously minor, but should we use this new model for the `domain` for sending emails (instead of the underlying `Site` model)?

Reply:
I think we shouldn't.

Reply:
Ok then!